packages/vscode-docker-registries/src/clients/RegistryV2/RegistryV2DataProvider.ts

```diff
@@ -19,6 +19,18 @@ export type V2Registry = CommonRegistry & V2RegistryItem;
 export type V2Repository = CommonRepository & V2RegistryItem;
 export type V2Tag = CommonTag & V2RegistryItem;
 
+interface ManifestHistory {

```

Comment:
Can you separate this manifest/createdAt stuff out into its own PR?

Reply:
yeah that's a good idea